aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,259 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AbbreviatedDateExample,
+  DateTimeNotificationExample,
+  RelativeTimeExample,
+  TimezoneExample,
+  ZeroPaddingExample,
+} from '../../examples';
+import { Text } from 'grommet';
+
+Date and time can appear in a wide variety of contexts:
+
+- Data tables
+- Inline with paragraph text
+- Name value lists
+- Notifications
+- Log files
+
+Depending on the context, date and time may be presented as:
+
+| Format                          | Purpose                                                                         | Examples                      | Notes                                                                              |
+| ------------------------------- | ------------------------------------------------------------------------------- | ----------------------------- | ---------------------------------------------------------------------------------- |
+| [Absolute time](#absolute-time) | To provide the precise date and/or time of an event.                            | September 15, 2023; 3/27/2023 | The granularity and formatting depends on the context. See absolute time guidance. |
```

Comment:
Should we be using the leading zero here? Also, should we be using Intl.DateTimeFormat() to present the dates?

```suggestion
| [Absolute time](#absolute-time) | To provide the precise date and/or time of an event.                            | September 15, 2023; 03/27/2023 | The granularity and formatting depends on the context. See absolute time guidance. |
```

Reply:
We can update to use `Intl.DateTimeFormat()`. But I think we are only recommending using leading zero if it's in a comparison, like in a column.

Reply:
Done.

Reply:
Commented somewhere else on this. but I believe we only want to use the zero padded in a column setting.

Reply:
Updated to use Date method.

Reply:
This really depends on the context here. Considering this is standalone in the table, I'd actually expect it to not have a leading zero. Might be easier to just put like 10/27/2023 instead to avoid confusion in this section.

Reply:
Done.